app/tests/actions/topics-test.js

```diff
@@ -5,8 +5,9 @@ import md5 from 'spark-md5';
 import { polyfill } from 'es6-promise';
 import axios from 'axios';
 import expect from 'expect';
-import * as actions from 'actions/topics';
-import * as types from 'types';
+import sinon from 'sinon';
```

Comment:
Good concerns @ZeroCho and @GGAlanSmithee . 

We aren't currently using webpack to retain the magic paths unfortunately. Although I find it really neat to do without nested `../../../`, I think in the future, I would favor moving the tests to within each component, similar to below:

```js
/component
   /__test__/
       - Component-test.js
   - Component.js
   - reducer.js
   - actions.js

```

My team and I have found it really neat and easy to locate components. Relative paths become less important when doing so. 

Hopefully, this doesn't become too much of a trade-off 🤞